code/src/index.css

```diff
@@ -1,13 +1,350 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+html {
+  height: 100%;
+}
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #161616;
+  color: #fff;
+  min-height: 100%;
 }
-
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
+}
+header {
+  width: 100%;
+  display: block;
+  border-bottom: 1px solid #252525;
+  padding: 10px 20px 10px 20px;
```

Comment:
We can make this line of code shorter - `padding: 10px 20px`

Why?

There are many CSS properties that gives us ability to apply some styles to each side of an element. Examples of those are `margin` and `padding`.

Order of applying those dimensions is:

1. `top`
2. `right`
3. `bottom`
4. `left`

### **Example 1: Apply 5px margin for all sides of an element**

We can be very specific when applying those values:

```css
.example {
  margin: 5px 5px 5px 5px;
}
```

We can however simplify this property, since all of them have the same value:

```css
.example {
  margin: 5px; // apply 5px for all 4 sides
}
```

### **Example 2: Apply 5px margin for vertical sides and 10px for horizontal sides of an element**

We can be very specific when applying those values:

```css
.example {
  margin: 5px 10px 5px 10px;
}
```

We can however simplify this property, since the sides have the same value:

```css
.example {
  margin: 5px 10px; // apply 5px for top & bottom and 10px for right & left sides
}
```